src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -50,6 +51,6 @@ public List<OrderResponseDto> getOrderHistory(Authentication auth) {
         return orderService.getOrdersHistory(user)
                 .stream()
                 .map(orderResponseDtoMapper::mapToDto)
-                .toList();
+                .collect(Collectors.toList());
```

Comment:
why do you need ArrayList instead of immutable?

Reply:
It requires to put JDK 16 or higher in pom file and if I do this I can't execute mvn clean package. I have got this problem since the begining Spring Security Unite. I think that this is a kind of problem with my settings in Idea but I can't solve it. Would you be so kind to give me advice with this? I'll try to realize it in next project.